src/app/game/game.component.ts

```diff
@@ -1,23 +1,38 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, HostListener, OnInit, OnDestroy } from '@angular/core';
+import { Subscription } from 'rxjs';
 
-import { Answer } from '../models/answer';
-import { AnswerService } from '../services/answer.service';
+import { GamePlayService } from '../services/game-play.service';
 
 @Component({
   selector: 'app-game',
   templateUrl: './game.component.html',
   styleUrls: ['./game.component.scss']
 })
-export class GameComponent implements OnInit {
-  protected answer: Answer = { id: 0, word: '' };
-  constructor(private answerService: AnswerService) { }
+export class GameComponent implements OnInit, OnDestroy {
+  constructor(private gameplayService: GamePlayService) { }
+  protected currentGuess = '';
+  protected acceptedGuesses: string[] = []
+  protected currentGuess$?: Subscription;
```

Comment:
Why is this `optional`? If you are getting errors from typescript about it not being defined, it's stand practice in Angular to use the non-null-assertion operator in a situation like this:

```Typescript
protected currentGuess$!: Subscription;
```